.github/workflows/run-tests.yml

```
@@ -112,16 +114,15 @@ jobs:
       - name: Mypy
         # Strict over the whole repository, with no module silenced by an
         # ignore_errors entry -- that list is empty. What remains is [tool.mypy]'s
-        # own configuration: the out-of-scope perf_test/ directory, two generated
-        # or vendored files, and ignore_missing_imports for the third-party
-        # packages that ship no types.
+        # own configuration: its exclude, for paths that are not project source,
+        # and ignore_missing_imports, for third-party packages that ship no types.
         # OPUS_CONFIG comes from the workflow-level env above.
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)